tests/test_api.py

```diff
@@ -332,43 +332,34 @@ async def test_api_update_resource(client, route, api_headers, api_headers_wrong
     }
 
     # Test API call with no token
-    resp = await client_http_method(path=route["url"], headers=None, json=payload)
+    resp = await client.put(path=f"/api/resources/{RESOURCE_ID}", headers=None, json=payload)
```

Comment:
Same here, just above in the diff, on line [311](https://github.com/datagouv/hydra/pull/132/files#diff-3451cc474d5698222168f66a293339a6bd296ee2c56f56409866bbae99917a79R311) you used the url WITH the final `/`, and here you use the same url WITHOUT the final `/`.

As long as it's on purpose i'm fine with it. I wonder if this would warrant a special parametrization, or maybe some tests dedicated to just that (using url with and without the final `/` just to make sure both work exactly the same).

Reply:
Same here, it's on purpose but since a previous merged PR, both work. See https://github.com/datagouv/hydra/pull/132/files#diff-e5ce70c8c6e9d9d078789d97ab169134b487c04923fa2b5b8bc584edee548e06R22